src/app/hooks/useBtcClient.ts

```diff
@@ -1,40 +1,21 @@
 import { BitcoinEsploraApiProvider } from '@secretkeylabs/xverse-core';
-import { useEffect, useMemo } from 'react';
+import { useMemo } from 'react';
 import useWalletSelector from './useWalletSelector';
 
 const useBtcClient = () => {
   const { network, btcApiUrl } = useWalletSelector();
   const { type, btcApiUrl: remoteBtcApiURL } = network;
+  const url = btcApiUrl || remoteBtcApiURL;
```

Comment:
what's the difference here between network.btcApiUrl and btcApiUrl

Reply:
The one from wallet selector is the URL if they already changed it to a custom value. The one on the network object is the default URL.